express/webhook.js

```diff
@@ -30,13 +33,26 @@ class WebhookRegistry {
         for (let [eventName, handlerData] of Object.entries(this._config.event_map)) {
             this._handlerMap[eventName] = handlerData;
         }
+
+        await this.getEventConfig();                                              // get event config for required event_map in eventConfig.event_configs
+        eventConfig.eventsMap = this._getEventIdMap(eventConfig.event_configs);   // generate eventIdMap from eventConfig.event_configs 
+        if (!this._validateEventsMap()) {                                         // validate eventIdMap for every element in _handlerMap 
+            throw new FdkWebhookRegistrationError(`Webhook config not found`);
```

Comment:
- Use invalid config error class. 
- Add event name which are not matched in validate function as part of error title.
- Enhance message here. Here we are checking event name details. Please change error message accordingly it being Webhook event not found. With details of dictionary key which is not found and version of it.